MetaMask/metamask-extension
# Remove the partially-applied dark mode styles used for beta

The beta build currently has a couple of UI elements (the horizontal logo and the app header) set to "dark mode", even though nothing else in our UI supports dark mode yet. These styles have been removed. We should make the entire UI support dark mode all at once at a later date.

Also the horizontal logo used for beta was named incorrectly; the dark and non-dark logos were swapped. They have been switched back so that the logo that ends with `-dark` is for dark mode.

Manual testing steps:  
  - Create a beta build using the command `yarn build dev --build-type beta`
  - See that the app header is not darkened, and that the correct horizontal logo is used in the top-left corner of the fullscreen UI.
